my_game.py

```diff
@@ -62,9 +62,8 @@ def __init__(self, filename, center_x, center_y, angle, speed, range, sound=None
 
         self.forward(self.speed)
 
-        # play the shot sound if present
-        if sound:
-            sound.play()
+    def destroy(self):
```

Comment:
We should not need this anymore, right? We can just use _kill()_ Was this not part of some workaround for the sound issue?

Reply:
That is correct